tests/acceptance/visual-routes-test.js

```diff
@@ -23,7 +23,7 @@ module('Acceptance | visual regression', function (hooks) {
     { title: '/index.html', route: '/'},
     { title: '/learn.html', route: '/learn/' },
     { title: '/legal.html', route: '/about/legal/' },
-    { title: '/logos.html', route: '/logos/' },
```

Comment:
you should not change the left hand side of this, if you do then we lose continuity. The left hand side is just the title for Percy to identify the snapshot by name. You can see that the left hand side (title) is already out of sync with the route in a few other places 👍

Reply:
that doesn't sound great tho :P

Reply:
it doesn't matter if it sounds great, it is how it's supposed to work :) it has been this way since the Ruby->Ember migration 👍 please fix